src/m1_facilitator/models/ERC20GatewayTokenPair.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
```

Comment:
These two lines(14, 15) are not needed in the license. 
https://github.com/mosaicdao/mosaic-1/blob/develop/contracts/erc20-gateway/ERC20Cogateway.sol#L3
https://github.com/mosaicdao/facilitator/blob/develop/src/m1_facilitator/repositories/Repositories.ts#L1

Reply:
Done.